packages/composables/src/composables/useShipping/index.ts

```diff
@@ -3,23 +3,32 @@ import {
   useShippingFactory,
   UseShippingParams
 } from '@vue-storefront/core';
-import type { ShippingAddress } from '@vue-storefront/vendure-api';
+import type { CreateAddressInput, Order, OrderAddress } from '@vue-storefront/vendure-api';
 import type {
   UseShippingAddParams as AddParams
 } from '../../types';
+import { useCart } from '../useCart';
 
-const params: UseShippingParams<ShippingAddress, AddParams> = {
+const params: UseShippingParams<OrderAddress, AddParams> = {
+  provide() {
```

Comment:
AFAIK `UseShippingParams` by default don't have `provide` function, are everything works well for TS?

Reply:
Yes, AFAIK everything is working correctly. No errors what so ever.

Reply:
Ok,  FactoryParams `shipps` provide fucntion